pkg/driver/node.go

```diff
@@ -751,7 +751,7 @@ func (d *nodeService) findDevicePath(devicePath, volumeID string) (string, error
 	}
 
 	// assumption it is a scsi volume for 3DS env
-	scsiName := "scsi-0QEMU_QEMU_HARDDISK_sd" + devicePath[len(devicePath)-1:]
+	scsiName := "scsi-0QEMU_QEMU_HARDDISK_sd" + devicePath[len(devicePath):]
```

Comment:
This fix will seems strange because I think the code are looking at disk named `scsi-0QEMU_QEMU_HARDDISK_sd`